src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import { getUser } from '../../api';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onModalToggle: (arg: boolean) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, onModalToggle }) => {
+  const { id: todoId, title: todoTitle, completed, userId } = todo;
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(userId).then(setUser);
```

Comment:
The `getUser` function call should include error handling to manage potential errors during the API call. Consider adding a `.catch` block to handle any errors that might occur.